### PR TITLE
Register/login functionality

### PR DESCRIPTION
**Pull request by nyazadzhiev**

### CODE REVIEW

File: server/controllers/Users.js [hunk 1]
```diff
@@ -45,7 +45,36 @@ usersController.post('/registration', async (req, res) => {
     })
 })
 
+usersController.get("/login", async (req, res) => {
+    const userData = {
+        username: req.body.username,
+        password: req.body.password
```
Comment (petya0111) [inline server/controllers/Users.js:28]: 1. you can extend the registration endpoint logic, for example for the field password to be validated with regex to have lowercase, uppercase, and special characters
2. resolve conflicts from the last merge

**Review by petya0111 — CHANGES_REQUESTED** [review]

File: server/controllers/Users.js [hunk 2]
```diff
@@ -45,7 +45,36 @@ usersController.post('/registration', async (req, res) => {
...
+    }
 
+    const user = await User.findOne({
+        where: {
+            email: userData.username
+        }
+    })
```
Comment (JordanRad) [inline server/controllers/Users.js]: This should be part of the service layer. Please, use UserService for that or create it if it's not initialised yet

File: server/server.js [hunk 3]
```diff
@@ -14,7 +19,25 @@ app.use(cors());
...
+const sessionConfig = {
+    secret: 'thisshouldbeabettersecret!',
+    resave: false,
+    saveUninitialized: true,
+    cookie: {
+        httpOnly: true,
+        expires: Date.now() + 1000 * 60 * 60 * 24 * 7,
+        maxAge: 1000 * 60 * 60 * 24 * 7
+    }
+}
```
Comment (JordanRad) [inline server/server.js]: Export this in a separate configuration in the ```/config``` folder. Use [this](https://github.com/Exordium-Project/e-shop/blob/dev/server/config/db.config.js) as a reference

**Review by JordanRad — CHANGES_REQUESTED** [review]

File: server/server.js [hunk 4]
```diff
@@ -14,7 +19,25 @@ app.use(cors());
 app.use(bodyParser.urlencoded({
     extended: false
 }));
+app.use(cookieParser())
 
+const sessionConfig = {
+    secret: 'thisshouldbeabettersecret!',
```
Comment (JordanRad) [inline server/server.js]: Please, set the secret in the .env file and read it from the file.

**Review by petya0111 — APPROVED** [review]

**Review by JordanRad — APPROVED** [review]

**Pull request merged**